test/runtests.jl

```diff
@@ -104,7 +104,7 @@ end
   Base.cumulative_compile_timing(true)
 end
 
-println("Making test list")
+@everywhere include("init_tests.jl")
```

Comment:
I think it makes sense to put this line in Serialization/runtests.jl and include the init_tests.jl in that folder as well. That is, if it still works? otherwise I would put at least put the init_tests.jl in the serialization folder

Reply:
Putting it into `serialization/runtests.jl` unfortunately does not work, as that is already its own parallel job (that might be on a different worker than the FTheoryTools test). Moving the file into the folder is no problem.